pkg/cmd/pipeline/describe_test.go

```diff
@@ -91,11 +91,11 @@ func TestPipelinesDescribe_with_run(t *testing.T) {
 	}
 
 	expected := []string{
-		"Name: pipeline",
+		"Name:   pipeline",
```

Comment:
Fixed to align with describing Pipelinerun! 

https://github.com/tektoncd/cli/blob/master/pkg/cmd/pipelinerun/describe.go#L33